tests/dependency-tests/root.rs

```diff
@@ -0,0 +1,285 @@
+use std::process::Command;
+
+#[derive(Debug)]
+enum Search<'a> {
+    Positive(&'a str),
+    Negative(&'a str),
+}
+
+#[derive(Debug)]
+struct Requirement<'a> {
+    human_readable_name: &'a str,
+    target: &'a str,
+    packages: &'a [&'a str],
+    features: &'a [&'a str],
+    default_features: bool,
+    search_terms: &'a [Search<'a>],
+}
+
+const ALL_WGPU_FEATURES: &[&str] = &[
+    "vulkan",
+    "gles",
+    "dx12",
+    "metal",
+    "webgpu",
+    "angle",
+    "vulkan-portability",
+    "webgl",
+    "noop",
+    "spirv",
+    "glsl",
+    "wgsl",
+    "naga-ir",
+    "strict_asserts",
+    "serde",
+    "static-dxc",
+    "counters",
+    "fragile-send-sync-non-atomic-wasm",
```

Comment:
note: Just calling out that this set actually changed, this isn't just a move. Looks like we added `vulkan` and `gles`, and reordered some things.

Reply:
I've replaced this with an explicit parse of cargo-metadata with `cargo_metadata` crate which makes it _super_ easy!